lib/requestWithOAuth2.js

```diff
@@ -5,24 +5,24 @@ var _ = require('underscore');
 
 var accessToken;
 
-function get(url, oauthUrl, clientId, clientSecret, username, password, callback) {
+function get(requestGetArgs, oauthUrl, clientId, clientSecret, username, password, callback) {
```

Comment:
As discussed request library calls it "options" or "params" - don't think either are better. target is a bit vague too